public/assets/images/icons/manageProjects/AccessDenied.tsx

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 
-function AccessDenied(props: any) {
+interface Props {
+  height?: string;
+  width?: string;
+}
+
+function AccessDenied(props: Props) {
```

Comment:
1. Since IconProps is defined, that can be used over here as well
2. Destructure the props